opentracing-jdbc/opentracing-jdbc/src/test/java/org/zalando/opentracing/jdbc/span/PatternPeerParserTest.java

```diff
@@ -0,0 +1,118 @@
+            "jdbc:hsqldb:mem:database,database",
+            "jdbc:h2:mem:database,database",
+            "jdbc:derby:target/tmp/derby/database;databaseName=database,target",
+            "jdbc:postgresql://localhost/database,localhost",
+            "jdbc:unsupported,",
+    })
+    void shouldParseHostname(final String url, final String expected) {
+        final Peer peer = unit.parse(url);
+        assertEquals(expected, peer.getHostname());
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+            "jdbc:oracle:thin:@db-users-production.team.svc.cluster.local:1521/database,",
+            "jdbc:mysql://db-users-production.team.svc.cluster.local/database,",
+            "jdbc:postgresql://db-users-production.team.svc.cluster.local/database,",
+            "jdbc:sqlserver://db-users-production.team.svc.cluster.local,",
+            "jdbc:mariadb://127.0.0.1/database,127.0.0.1",
+            "jdbc:db2://127.0.0.1:50000/database,127.0.0.1",
+            "jdbc:sap://db-users-production.team.svc.cluster.local:30015/,",
+            "jdbc:informix-sqli://127.0.0.1:9088/sysuser:INFORMIXSERVER=database,127.0.0.1",
+            "jdbc:hsqldb:mem:database,",
+            "jdbc:h2:mem:database,",
+            "jdbc:derby:target/tmp/derby/database;databaseName=database,",
+            "jdbc:postgresql://localhost/database,",
+            "jdbc:unsupported,",
+    })
+    void shouldParseIpv4(final String url, final String expected) {
+        final Peer peer = unit.parse(url);
+        assertEquals(expected, peer.getIpv4());
+        assertNull(peer.getIpv6());
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+            "jdbc:oracle:thin:@db-users-production.team.svc.cluster.local:1521/database,",
+            "jdbc:mysql://db-users-production.team.svc.cluster.local/database,",
+            "jdbc:postgresql://db-users-production.team.svc.cluster.local/database,",
+            "jdbc:sqlserver://db-users-production.team.svc.cluster.local,",
+            "jdbc:mariadb://[2001:0db8:85a3:0000:0000:8a2e:0370:7334]/database,2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+            "jdbc:db2://[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:50000/database,2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+            "jdbc:sap://db-users-production.team.svc.cluster.local:30015/,",
+            "jdbc:informix-sqli://[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:9088/sysuser:INFORMIXSERVER=database,2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+            "jdbc:hsqldb:mem:database,",
+            "jdbc:h2:mem:database,",
+            "jdbc:derby:target/tmp/derby/database;databaseName=database,",
+            "jdbc:postgresql://localhost/database,",
+            "jdbc:unsupported,",
+    })
+    void shouldParseIpv6(final String url, final String expected) {
+        final Peer peer = unit.parse(url);
+        assertEquals(expected, peer.getIpv6());
+        assertNull(peer.getIpv4());
+    }
+
+    @ParameterizedTest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/tracer/pullRequest?prid=4563925)